nitrates/llh_analysis/do_manage2.py

```diff
@@ -1182,6 +1264,38 @@ def get_outFoVmap_inds(
     return good_map, good_hp_inds
 
 
+def get_sao_file(trigtime):
```

Comment:
why are you doing this here instead of in the data setup?

Reply:
I think the sao file is always available? it's necessary for th eprocessing the SDC does.
Unless this is a real issue, i think it should be done during setup.

Reply:
I'm not 100% sure that this is a real issue. It's that the sao file (at least on roar collab) is being grabbed differently than the other data, which is being constantly downloaded. Here the sao file is using the API, which might be behind in time the other method of constantly grabbing data.

Reply:
If I were to move this to setup time, I would most likely have it move on if it can't find the sao file, and then try again here to find it if it wasn't previously found.

Reply:
I dont think the SDC is able to even post any data files without having the sao file, so i dont think this will be a problem... 
Have you ever not found an sao file?